src/test/java/myfirstmodule/ApplicationRootUrlTest.java

```diff
@@ -0,0 +1,18 @@
+package myfirstmodule;
+
+import com.mendix.core.Core;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ *
+ * @author reinout
+ */
+public class ApplicationRootUrlTest extends MendixUnitTestBase {
+
+    @Test
+    public void testApplicationRootURL() {
+        assertEquals("http://localhost:8080", Core.getConfiguration().getApplicationRootUrl());
```

Comment:
Perhaps we should add an error message to the `assertEquals` so that in case of failure, it is clear that `m2ee.yaml` should be checked, as the test case fails or succeeds based on this external input.

Reply:
Done